.github/actions/setup-build/action.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 The StableHLO Authors.
+
+# The setup-build action gets everything needed by buildAndTest into the workspace.
+# This checks out the proper revision of the LLVM repository, installs ninja, and sets
+# up the ccache for faster re-compilation.
+name: "Setup build environment (ninja, ccache, llvm, lld)"
+
+inputs:
+  llvm-ref:
+    description: |
+      LLVM Commit Ref for checkout and build. Used for ccache value and checkout.
+    required: true
+
+
```

Comment:
How many newlines are idiomatic to separate sections in YAML files? I skimmed some .yml files, and I don't think it's more than 1.

Reply:
Deleted additional new line. I tried some searching for YMAL conventions and it looks like a mix of 1 line and 0 lines.